syntax = "proto3";
package value;

message AddService
{
    int32 number1 = 1;
    int32 number2 = 2;

    int32 result = 3;
}

message Any
{
    bytes data = 1;
}